tests/openvino/utils_tests.py

```diff
@@ -137,6 +137,7 @@
     "xglm": "hf-internal-testing/tiny-random-XGLMForCausalLM",
     "xverse": "katuni4ka/tiny-random-xverse",
     "glm4": "katuni4ka/tiny-random-glm4",
+    'open-clip': 'laion/CLIP-ViT-L-14-laion2B-s32B-b82K'
```

Comment:
2Gb, still big for testing, can we try this one (it is smallest what I found https://huggingface.co/apple/MobileCLIP-S1-OpenCLIP)?

Reply:
done